pyproject.toml

```diff
@@ -54,9 +56,6 @@ match_dir = "(datamodules|datasets|losses|models|samplers|torchgeo|trainers|tran
 addopts = "-m 'not slow'"
 # https://docs.pytest.org/en/latest/how-to/capture-warnings.html
 filterwarnings = [
-    # Treat all warnings as errors
-    "error",
```

Comment:
Took me 2 whole days to figure out that this is why the tests started failing all of a sudden after the rebase. I think this is too strict. Users running `pytest` from the command line will still see a list of warnings at the bottom. We only need to convert that to an error during CI.